build.gradle

```
@@ -50,7 +40,7 @@ static def getDate() {
 def build = System.env.BUILD_NUMBER == null ? "" : "-${System.env.BUILD_NUMBER}"
 
 group = "org.key-project"
-version = "2.13.0$build"
+version = "2.12.0$build"
```

```suggestion
version = "2.13.0$build"
```
We need to revert this, otherwise we would have the wrong version number on main, right?